opm/upscaling/writeECLData.hpp

```diff
@@ -23,6 +23,8 @@
 
 #include <string>
 
+#include <time.h>
```

Comment:
Shouldn't this have been `#include ctime` as we are using C++?

Reply:
> Shouldn't this have been `#include ctime` as we are using C++?

Actually, no.  The function signature uses `time_t`, not `std::time_t`.  We could **arguably** write this as
```C++
#include <ctime>

using std::time_t;
```
but we get that for free by using the C header instead.

Reply:
That said, I suppose we could update the signature to `std::time_t`, but i didn't want to track down all callers and fix those too.